yarn.lock

```diff
@@ -14178,16 +14218,14 @@ react-native-svg@^12.1.1:
     css-select "^2.1.0"
     css-tree "^1.0.0-alpha.39"
 
-react-native-test-app@^0.3.2:
-  version "0.3.13"
-  resolved "https://registry.yarnpkg.com/react-native-test-app/-/react-native-test-app-0.3.13.tgz#ec4cd2d35110b4dbb079b4df91fb8127af359bb6"
-  integrity sha512-b+QoP3Wt+2XKvwQx1l6OqF17KhBQOvFoj+WVGNzFs+gVDumNUInFdzjKXo4G3ZTXJqTmGoBKYRpIuWEKoNJK9Q==
+react-native-test-app@^0.5.9:
+  version "0.5.14"
```

Comment:
Not using the latest 0.5.x due to a crash on startup. The fix for that is here: https://github.com/microsoft/react-native-test-app/pull/370